src/utils/internalState.ts

```diff
@@ -0,0 +1,17 @@
+import { ContextProps } from './formContext';
+
+type InternalState = {
+  formState: {
```

Comment:
probably should drop the formState prop? Since init is already global, I guess things here are likely to be tied to a specific form so maybe makes since for the UUIDs to be the primary key for this obj

Reply:
yeah agreed